pkg/test/framework/components/environment/native/native.go

```diff
@@ -30,7 +30,7 @@ import (
 
 const (
 	systemNamespace = "istio-system"
-	domain          = "cluster.local"
+	domain          = "Cluster.local"
```

Comment:
huh?

Reply:
@howardjohn I assume this still needs to be fixed?

Reply:
Yeah working on this, moving some more tests over once I saw I need 4 different approvals :slightly_smiling_face: